lib/node_modules/@stdlib/lapack/base/dlaswp/docs/repl.txt

```diff
@@ -0,0 +1,130 @@
+
+{{alias}}( order, N, A, LDA, k1, k2, IPIV, sipiv )
```

Comment:
```suggestion
{{alias}}( ord, N, A, LDA, k1, k2, IPIV, sipiv )
```

Is this fine, or we shall keep it `ord`?

Reply:
Either. The main motivation for abbreviating to `ord` is for long function signatures where we need to shave off characters to ensure fits within an 80 character line length.